config/main.py

```diff
@@ -3252,6 +3253,9 @@ def reload(ctx, no_dynamic_buffer, no_delay, dry_run, json_data, ports, verbose)
     if buffer_model_updated:
         print("Buffer calculation model updated, restarting swss is required to take effect")
 
+    if not status:
```

Comment:
Add error log with additional details

Reply:
Previously in _wait_until_clear(), it has printed with this error, maybe user can refer to this one😊
+    if not empty and timeout:
         click.echo("Operation not completed successfully, please save and reload configuration.")